### PR TITLE
Add get file paths method

### PR DESCRIPTION
**Pull request by nehaabrol87**
@jonathanbarton PR for adding the getFilePaths method

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,15 +12,32 @@ function getJsonFilesFromFolder(fileFolder) {
 		var fileContent = fs.readFileSync(`${__dirname}/${file}`);
 		filesObject[basename] = JSON.parse(fileContent);
 	});
-	return filesObject; 
+	return filesObject;
+}
+
+function getFilePaths(type) {
+  var validTypes = ['seeds'];
+  var isValidType = validTypes.indexOf(type) > -1 ? true : false;
```
Comment (jonathanbarton) [inline index.js]: this just needs to be:
`
var isValidType = (validTypes.indexOf(type) > -1);
`

Comment (nehaabrol87) [reply index.js]: haha ofcourse.I am stupid.

File: index.js [hunk 2]
```diff
@@ -12,15 +12,32 @@ function getJsonFilesFromFolder(fileFolder) {
...
+
+  if (isValidType) {
+    var files = ['flights', 'telemetry'];
+    var filePaths = [];
+
+    files.forEach(function(file) {
```
Comment (jonathanbarton) [inline index.js]:
```
var filePaths = files.map(function(file){ 
  return `${__dirname}/static/${type}/${file}.json`; 
});
```

Comment (jonathanbarton) [reply index.js]: sorry - weird markdown issue, ^^ this

**Comment by jonathanbarton** [comment]
lgtm otherwise!

File: index.js [hunk 3]
```diff
@@ -12,15 +12,31 @@ function getJsonFilesFromFolder(fileFolder) {
...
+function getFilePaths(type) {
+  var validTypes = ['seeds'];
+  var isValidType = validTypes.indexOf(type) > -1;
+
+  if (isValidType) {
+    var files = ['flights', 'telemetry'];
+    var filePaths = [];
+
+    var filePaths = files.map(function(file) {
+      return `${__dirname}/static/${type}/${file}.json`;
```
Comment (nehaabrol87) [inline index.js:27]: @jonathanbarton  Changes incorporated. Review please.

**Comment by jonathanbarton** [comment]
👍

**Pull request merged**